src/Benchmarks.ClientJob/project.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": "1.0.0-*",
+  "description": "Class to transfer data to/from the benchmark client.",
+  "repository": {
```

Comment:
Note: version/description/repository sections aren't really necessary for test/sample/tool projects that we aren't creating packages for. Doesn't hurt though.